CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+# 1.1.5 - 09.04.2024
+
+- Adapt code for PhpStorm 2024.1
```

Comment:
`.run/Run Tests.run.xml `I haven't changed it

Reply:
Done